src/main/java/com/xiao/users/controller/UserController.java

```diff
@@ -0,0 +1,45 @@
+package com.xiao.users.controller;
+
+import com.xiao.users.dto.ResponseDto;
+import com.xiao.users.dto.UserDto;
+import com.xiao.users.service.IUserService;
+import jakarta.validation.Valid;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
```

Comment:
Let's define a base mapping
Afterward, let's ensure that all resource names adhere to the RESTful API standards.

Please refer to this documentation: https://restfulapi.net/resource-naming/
@tnd-dev99